docker-compose.prod.yml

```diff
@@ -0,0 +1,22 @@
+services:
+  backend:
+    build:
+      context: ./backend
+      dockerfile: Dockerfile
+    ports:
+      - "8000:8000"
```

Comment:
to też nie powinno być wystawione poza app-network skoro do backendu dobijamy się przez nginx